src/components/more-from-blogs/more-from-blogs.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../card";
+
+import * as styles from './more-from-blogs.module.scss';
+
+const MoreFromBlogs = ({ data, refMoreFrom, moreFromAuthor }) => {
```

Comment:
I think the name of the prop `moreFromAuthor` isn't easy to understand what is the meaning of this prop

Reply:
This prop set the text depending on what author of more blogs from I have, so, I have two cases: More from xmartlabs and more from [author name].

Reply:
Done